fieldservice_sale/security/ir.model.access.csv

```diff
@@ -1,2 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_sale_order_fsm_user,sale.order.fsm.user,model_sale_order,fieldservice.group_fsm_user,1,0,0,0
+access_sale_order_template_portal,access.sale.order.template.portal,sale_management.model_sale_order_template,base.group_portal,1,0,0,0
```

Comment:
We don't deal with the sale order template model in fieldservice so why are we adding this here?

Reply:
@brian10048 I got a permission error in the mobile app for the portal group.